File: 1 - metodosBusca/pacoteBusca_JomiHubner/src/grafo/ArestaNaoDirigidaValorada.java
package grafo;

public class ArestaNaoDirigidaValorada extends ArestaValorada {
	
	public ArestaNaoDirigidaValorada(Vertice vi, Vertice vj, int custo) {
		super(vi, vj, custo);
		vi.addAresta(this);
		vj.addAresta(this);
	}
}

File: 1 - metodosBusca/python/app/core/grafo/aresta_nao_dirigida_valorada.py
from .aresta_valorada import ArestaValorada

class ArestaNaoDirigidaValorada(ArestaValorada):
    def __init__(self, vi, vj, custo):
        super().__init__(vi, vj, custo)
        vi.add_aresta(self)
        vj.add_aresta(self)